pandas/core/frame.py

```diff
@@ -1046,16 +1046,71 @@ def __len__(self):
 
     def dot(self, other):
         """
-        Matrix multiplication with DataFrame or Series objects. Can also be
-        called using `self @ other` in Python >= 3.5.
+        Compute the matrix mutiplication between the DataFrame and other.
+
+        This method computes the matrix product between the DataFrame and the
+        values of an other Series, DataFrame or a numpy array.
+
+        It can also be called using `self @ other` in Python >= 3.5.
```

Comment:
```suggestion
        It can also be called using ``self @ other`` in Python >= 3.5.
```

sorry I didn't see that before, for code we use double backticks (single backticks are for things we reference, like a function name, a class, a variable...)